README.md

```diff
@@ -117,9 +88,7 @@ After you're finished please remove all the comments and instructions!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://last-3f0w.onrender.com/users/1/groups)
```

Comment:
- [ ] You app doesn't show up :broken_heart: Please make sure to fix your deployment and make your app works as expected :+1:

Reply:
- The link https://last-3f0w.onrender.com throws the error `Not found`. Please fix that